Aggiunge capitolo "come affrontare l'età nel mondo it" (#157)
Il-Libro-Open-Source/book: docs/it/come-affrontare-l-eta-nel-mondo-it.md

> # Come affrontare l'età nel mondo IT

Author: Perché credi crei una percezione negativa?
In realtà il titolo è atto ad attirare una persona a incuriosirsi sull'argomento e leggere che quello che lui percepisce come "Una sfida da affrontare" in realtà è un'opportunità sotto vari aspetti.

> # Come affrontare l'età nel mondo IT

Reviewer: Esatto, è proprio questo il fulcro della situazione: non c'è nulla da affrontare, quanto più prendere coscienza della realtà che invecchiare fa parte della vita. Se la mettiamo sotto la prospettiva della sfida, vuol dire che c'è chi vince e chi perde, ma qui non vince né perde nessuno a mio avviso.

Non so se "gestire" sia il termine più appropriato

> In questo capitolo affronteremo il tema dell'età nel mondo IT, un tema che spesso viene sottovalutato ma che è molto importante per chi lavora in questo settore e che spesso si trova a dover fare i conti con la propria età chiedendosi se è ancora competitivo sul mercato del lavoro, se è ancora in grado di stare al passo con le nuove tecnologie o se invece è costretto a districarsi tra ruoli manageriali e non più tecnici.

Author: La butto lì, dimmi se ti suona meglio:
In questo capitolo esploreremo la questione dell'età nel contesto dell'ambito IT. Spesso trascurato, questo aspetto riveste notevole rilevanza per coloro che operano in questo settore da qualche anno. Molte persone si trovano a confrontarsi con l'interrogativo sulla propria competitività nel mercato del lavoro in relazione all'età, chiedendosi se sono ancora in grado di mantenere un livello di competenza adeguato alle nuove tecnologie o se, viceversa, sono destinati a navigare tra mansioni manageriali, abbandonando il ruolo tecnico.

> In questo capitolo affronteremo il tema dell'età nel mondo IT, un tema che spesso viene sottovalutato ma che è molto importante per chi lavora in questo settore e che spesso si trova a dover fare i conti con la propria età chiedendosi se è ancora competitivo sul mercato del lavoro, se è ancora in grado di stare al passo con le nuove tecnologie o se invece è costretto a districarsi tra ruoli manageriali e non più tecnici.

Reviewer: Suona decisamente meglio! Grazie!

> E' opportuno precisare che l'obiettivo di questo capitolo non è di certo dare delle certezze assolute o definire percorsi di carriera uguali per tutti, in quanto ogni persona è unica e diversa in termini di passioni, ambizioni e attitudini. Faremo infatti delle considerazioni personali che potrebbero aiutarti a compredere come vuoi vivere la tua carriera dopo molti anni di esperienza e riflettere sulle implicazioni che potrebbero comportare alcune scelte.

Reviewer 2: ```suggestion
E' opportuno precisare che l'obiettivo di questo capitolo non è di certo dare delle certezze assolute o definire percorsi di carriera uguali per tutti, poiché ogni persona è unica e diversa in termini di passioni, ambizioni e attitudini. Faremo quindi delle considerazioni personali che potrebbero aiutarti a compredere come vuoi vivere la tua carriera dopo molti anni di esperienza e riflettere sulle implicazioni che potrebbero comportare alcune scelte.
```

## Pensare al "vero" obiettivo

Ripetiamo: il lavoro del developer è (anche) risolvere problemi. Il nostro scopo non è, e non sarà sempre, trovare soluzioni accativanti, magari con le tecnologie del momento. Ed è questa la maturità che deve acquisire un developer negli anni e con l'esperienza.

> Vogliamo evidenziare questo aspetto con un aneddoto riportato nel libro Clean Architecure di Robert C. Martin. Senza entrare troppo nei dettagli, che puoi trovare nel libro, durante una sua esperienza lavorativa alla fine degli anni Ottanta ha dovuto, per pura esigenza di marketing, impiegare un database relazionale in un sistema di cui si occupava. Ma per motivi ingegneristici lui affermava che non era assolutamente necessario avere un database in quel sistema, che raggiungeva appieno il suo scopo così com'era, ma il mercato di quel periodo voleva i database nei sistemi, e l'azienda _accontentò_ il mercato. Dal canto suo, Martin era focalizzato sull'obiettivo ingegneristico reale, e crediamo che debba essere questa la competenza che chi fa questo lavoro dovrebbe sviluppare col tempo.

Reviewer 3: Ciao!
Provo a chiarire questo concetto: è inteso che come developer (che sviluppa per portare valore e risolvere problemi di business) non era interessato aintrodurre nel sistema la tecnologia del momento (i db in quel periodo) pur di usarla, ma era interessato semplicemente ad avere un'architettura software "adatta" al problema e alle esigenze dell'azienda, quindi focalizzando si sull'obiettivo ingegneristico reale. 
L'aneddoto vuole solo evidenziare ciò che viene detto all'inizio: "Il nostro scopo non è, e non sarà sempre, trovare soluzioni accativanti, magari con le tecnologie del momento. Ed è questa la maturità che deve acquisire un developer negli anni e con l'esperienza."

> Vogliamo evidenziare questo aspetto con un aneddoto riportato nel libro Clean Architecure di Robert C. Martin. Senza entrare troppo nei dettagli, che puoi trovare nel libro, durante una sua esperienza lavorativa alla fine degli anni Ottanta ha dovuto, per pura esigenza di marketing, impiegare un database relazionale in un sistema di cui si occupava. Ma per motivi ingegneristici lui affermava che non era assolutamente necessario avere un database in quel sistema, che raggiungeva appieno il suo scopo così com'era, ma il mercato di quel periodo voleva i database nei sistemi, e l'azienda _accontentò_ il mercato. Dal canto suo, Martin era focalizzato sull'obiettivo ingegneristico reale, e crediamo che debba essere questa la competenza che chi fa questo lavoro dovrebbe sviluppare col tempo.

Reviewer 4: Ciao, sulla menzione è necessario per il lettore sapere da che parte arrivano i dati appunto per avere la possibilità di farsi usa sua idea se necessario, le fonti le trovo necessarie e corrette nei confronti del lettore; risultati e frasi, vengono riportati così come sono e non sono frutto di rielaborazioni. Sul fatto che sia aleatorio il risultato trovo sia il contrario visto che è citato un articolo sottoposto a peer review e non in stato di valutazione, è citato e altri studi si basano su di esso, mi sembra di base solida.

Facciamo un esempio più recente: i microservizi. Chissà in quanti annunci di lavoro li avrai visti in questi ultimi anni.
Al di là dei vantaggi (e svantaggi) di questa architettura, sembra che non se ne possa più fare a meno. Ma nel marzo 2023, il team di Prime Video ha pubblicato un post sul proprio blog, indicando che con il passaggio da un'architettura distribuita a microservizi a un'architettura monolitica ha ridotto i costi infrastrutturali di oltre il 90%. Il team di Prime Video si è focalizzato su un obiettivo preciso: ottimizzare i costi. E ci sono riusciti con una soluzione tanto semplice quanto _classica_, il monolite.

I due esempi sono diversi in termini di situazione e tempi, ma ci fanno comprendere una cosa: il nostro lavoro è cercare il modo migliore di raggiungere l'obiettivo ingegneristico che aiuti il business, a prescindere dalle mode del momento.

Se sei un developer che riconosce del valore nelle scelte che compie, perché preoccuparsi dell'età che avanza?

## Diventare manager

> Facciamo una breve considerazione sul diventare manager in modo da giungere alle dovute riflessioni che potrebbero esserti utili semmai un giorno dovessi fare questa scelta. Per manager intendiamo, per semplicità e senza andare troppo nel dettaglio, quei ruoli che non sono coinvolti nelle decisioni, e operativi per quanto concerne l'aspetto tecnico ma sostanzialmente incentrati nella gestione di alto livello dei progetti e dei team.

Reviewer 3: Ciao!
La frase è solo per soffermarsi brevemente che diventare manager vuol dire che quasi sicuramente non si toccherà più codice e tantomento si è coinvoloti su decisioni architetturali, ma ci saranno altre mansioni: gestire budget, ferie, mebri del team, tempistiche progetti (no stime) ecc..

Ma rileggendo bene il contesto, potrebbe sembrar scontato questo concetto e si potrebbe pensare di eliminare la frase. Lascio ovviamente a voi la decisione finale.

> Facciamo una breve considerazione sul diventare manager in modo da giungere alle dovute riflessioni che potrebbero esserti utili semmai un giorno dovessi fare questa scelta. Per manager intendiamo, per semplicità e senza andare troppo nel dettaglio, quei ruoli che non sono coinvolti nelle decisioni, e operativi per quanto concerne l'aspetto tecnico ma sostanzialmente incentrati nella gestione di alto livello dei progetti e dei team.

Author: Credo possa essere scontato, ma vale la pena ricordarlo.

Abbiamo constatato che è possibile rimanere dev anche con l'avanzare dell'età, ma molto spesso diventare manager sembra quasi l'unico modo per fare carriera. Ci sono realtà aziendali con una gerarchia tale per cui chi fa sviluppo è giunto solo ad un primo step della propria carriera, una base dalla quale poter avanzare, prima o poi, abbandonando la via del codice a favore di quella del manager. Noi ci auguriamo che questo non sia il tuo caso, ma può capitare che il contesto lavorativo in cui ci si trova influenzi il modo di percepire la carriera e di conseguenza le nostre scelte, perché vediamo un'unica strada davanti a noi. Questo potrebbe indurci a pensare che non raggiungere un ruolo manageriale dopo molti anni di esperienza equivalga a un fallimento di carriera, portando noi stessi a sentirci in quel momento inadeguati rispetto alle richieste del mercato. Per evitare ciò, si sceglie di diventare manager per raggiungere una sorta di traguardo professionale e, perchè no, anche salariale.

> Caro lettore, vogliamo dirti che non è così: se in te c'è passione per lo sviluppo non sei costretto a intraprendere percorsi manageriali. La programmazione offre molte sfide e soddisfazioni che puoi trovare in contesti diversi da quello nel quale probabilmente ti trovi attualmente. Continua a studiare, ad approfondire e a confrontarti con altre persone con la tua stessa passione. E se dovessi realizzare di non trovarti bene nel contesto attuale, cerca realtà aziendali dove ci sono percorsi alternativi, che ti permetteranno di crescere tecnicamente e di continuare a perseguire i tuoi obiettivi.

Reviewer 5: ```suggestion
A te che leggi, vogliamo dire che non è così: se in te c'è passione per lo sviluppo, non hai l'obbligo di intraprendere percorsi manageriali. La programmazione offre molte sfide e soddisfazioni che puoi trovare in contesti diversi da quello nel quale probabilmente ti trovi attualmente. Continua a studiare, ad approfondire e a confrontarti con altre persone con la tua stessa passione. E se dovessi realizzare di non trovarti bene nel contesto attuale, cerca realtà aziendali dove ci sono percorsi alternativi, che ti permetteranno di crescere tecnicamente e di continuare a perseguire i tuoi obiettivi.
```

Concludiamo questa sezione dicendoti che non vogliamo convincerti a non diventare manager. Ci sono tante sfide in questo ruolo che sicuramente ti darebbero molte soddisfazioni. Diventa manager, abbandonando pure la parte tecnica e il codice, se ti rivedi in questa figura, se questo ruolo ti appassiona e sei convinto di questa decisione. Siamo certi che se intraprenderai questo percorso con decisione e passione potrai diventare un manager con il quale tutti avranno piacere di collaborare.

## Non _solo_ developer

In questa sezione rifletteremo brevemente su una domanda in apparenza semplice ma allo stesso tempo interessante: se desideriamo continuare a scrivere codice, è necessario rimanere developer per sempre?

> Probabilmente nella tua mente starai già pensando al tuo contesto aziendale o alle tue precedenti esperienze, cercando figure _ibride_ tra ruoli tecnici e manageriali: per citarne alcune ci sono figure come Team Leader, Tech Lead e Solution Architect a cui i developers fanno riferimento per ciò che riguarda decisioni più o meno delicate su aspetti tecnici. Come forse saprai, non è detto che queste figure, prese da altre responsabilità o coinvolte in altri processi più organizzativi, riescano ancora effettivamente a scrivere codice. Ogni realtà solitamente ridefinisce questi ruoli, proiettandoli nei propri contesti. Per esempio, un team leder in una start-up che è poco strutturata è probabile che continui a sviluppare, rispetto a realtà più grandi e strutturate dove il team leader è più simile ad una figura manageriale. Ci sono, invece, realtà molto flessibili che danno la libertà ai propri team leader di gestirsi e decidere quanto addentrarsi negli aspetti tecnici.

Reviewer: ```suggestion
Probabilmente nella tua mente starai già pensando al tuo contesto aziendale o alle tue precedenti esperienze, ricostruendo mentalmente le responsabilità di figure _ibride_ tra ruoli tecnici e manageriali, per citarne alcune: Team Leader, Tech Lead e Solution Architect a cui i developers fanno riferimento per ciò che riguarda decisioni più o meno delicate su aspetti tecnici. Come forse saprai, non è detto che queste figure, prese da altre responsabilità o coinvolte in altri processi più organizzativi, riescano ancora effettivamente a scrivere codice. Ogni realtà solitamente ridefinisce questi ruoli, proiettandoli nei propri contesti. Per esempio, un team leder in una start-up che è poco strutturata è probabile che continui a sviluppare, rispetto a realtà più grandi e oliate dove il team leader è più simile ad una figura manageriale. Ci sono, invece, realtà molto flessibili che danno la libertà ai propri team leader di gestirsi e decidere quanto addentrarsi negli aspetti tecnici.
```

Questi ruoli sono molto interessanti e sfidanti qualora fare _solo coding_ iniziasse a non bastarti. Se vuoi gestire un team e avere responsabilità differenti, sicuramente prima o poi potrai intraprendere uno di questi percorsi senza rinunciare del tutto all'aspetto tecnico.

> Al di là della tua scelta, ricorda che se vuoi essere un developer che aggiunga valore nonostante l'età, non è sufficiente che ti limiti al coding, cerca sempre di contribuire a ogni aspetto del progetto: sia che tu rivesta il ruolo di developer o di team leader, impegnati costantemente nel risolvere problemi e suggerire soluzioni ottimali, senza concentrarti sui limiti assegnati dai ruoli.

Reviewer: ```suggestion
Al di là della tua scelta, ricorda che se vuoi essere un developer che aggiunga valore nonostante l'età, non è sufficiente che ti limiti al coding, cerca sempre di contribuire a ogni aspetto del progetto: sia che tu rivesta il ruolo di developer o di team leader, impegnati costantemente nel risolvere problemi e suggerire soluzioni, senza farti distrarre dai limiti imposti dai ruoli.
```

> Se sei ancora junior, proponi soluzioni anche se non sei del tutto sicuro della loro correttezza. Non solo l'impegno nello studio e nell'approfondimento, ma anche e soprattutto trarre insegnamento dai propri errori è un ottimo modo per crscere.

Reviewer 5: ```suggestion
Se sei ancora junior, proponi soluzioni anche se non hai la certezza in merito alla loro correttezza. Non solo l'impegno nello studio e nell'approfondimento, ma anche e soprattutto trarre insegnamento dai propri errori è un ottimo modo per crescere.
```

Così, dopo anni di esperienza, potrai mettere in campo molte conoscenze che figure più giovani del team probabilmente non avranno.

> Un developer che sa comunicare e sa trasmettere le proprie conoscenze ad altri, sarà sempre di grande valore per un team e avrà un'autorevolezza al suo interno a prescindere dal ruolo e soprattutto dall'età.

Reviewer 5: ```suggestion
Un/a developer che sa comunicare e sa trasmettere le proprie conoscenze ad altri, sarà sempre di grande valore per un team e avrà un'autorevolezza al suo interno a prescindere dal ruolo e soprattutto dall'età.
```

## Il bias del fare codice a vita

> La carriera di chi sviluppa è complessa da molti punti di vista e non priva di incognite e, al primo posto, possiamo mettere il fatto che sia una categoria nuova e in continua evoluzione. Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante. Cercando di allontanarsi un poco per vedere un percorso di massima non possiamo non pensare alla crescita verticale verso mansioni mano a mano più gestionali e meno operative, è un pensiero che prima o poi si tende a fare. Questo è legato non solo all’ambizione ma anche ad un pensiero sul proprio futuro, in generale sull’invecchiamento intellettuale e sulle conseguenze che inevitabilmente può avere sul nostro rendimento, sulla nostra freschezza e sulla capacità di stare dietro ad un mondo, quello informatico, in perenne vorticoso movimento; questo rendimento è legato a doppio filo con il nostro intelletto.

Reviewer: Riuscireste a rielaborare questa frase? A mio avviso non è molto scorrevole.

"Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante"

```suggestion
La carriera di chi sviluppa è complessa sotto molti punti di vista e non priva di incognite: al primo posto possiamo assolutamente prendere coscienza del fatto che sia una categoria nuova e in continua evoluzione. Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante. Da un punto di vista più generale non possiamo ignorare la crescita verticale verso determinate mansioni più gestionali e meno operative, è un pensiero che prima o poi si tende a fare. Questo è legato non solo all’ambizione ma anche ad una considerazione sul proprio futuro, in generale sull’invecchiamento intellettuale e sulle conseguenze che inevitabilmente può avere sul nostro rendimento, sulla nostra freschezza e sulla capacità di stare dietro ad un mondo, quello informatico, in perenne vorticoso movimento; questo rendimento è legato a doppio filo con il nostro intelletto.
```

Author: Non committo per non perdere il tuo commento.. Dimmi se il re-wording ti piace:
Guardando alcuni schemi che illustrano l'evoluzione e le competenze necessarie per una particolare mansione, diventa evidente di fronte a noi un albero di opzioni impressionante.

Reviewer: E' proprio il concetto di albero che secondo me può creare confusione. Ho capito che fai riferimento a un albero decisionale, ma ho paura che non possa essere così scontato agli occhi di altri.

Se usassimo in qualche modo le parole "diramazioni" o "strade"?

Reviewer 4: @AngeloAvv essendo un libro per DEV io credo che il concetto di albero sia vicino all'interlocutore

Reviewer: @eppak un wannabe dev o un junior se non ha studiato algoritmi e strutture dati in modo approfondito è difficile che conosca cosa sia un albero

Author: @AngeloAvv però il capitolo non è improntato a uno wannabe dev.. credo possa essere sensato fare congetture di questo tipo.

Reviewer: Vero, ma questo non esclude che possa volerlo comunque leggere per capire cosa aspettarsi dal futuro. Se volete lasciarlo così per me va bene la tua seconda versione @Cadienvan

> La carriera di chi sviluppa è complessa sotto molti punti di vista e non priva di incognite: al primo posto possiamo assolutamente prendere coscienza del fatto che sia una categoria nuova e in continua evoluzione. Guardando alcuni schemi che illustrano l'evoluzione e le competenze necessarie per una particolare mansione, diventa evidente di fronte a noi un albero di opzioni impressionante. Da un punto di vista più generale non possiamo ignorare la crescita verticale verso determinate mansioni più gestionali e meno operative, è un pensiero che prima o poi si tende a fare. Questo è legato non solo all’ambizione ma anche ad una considerazione sul proprio futuro, in generale sull’invecchiamento intellettuale e sulle conseguenze che inevitabilmente può avere sul nostro rendimento, sulla nostra freschezza e sulla capacità di stare dietro ad un mondo, quello informatico, in perenne vorticoso movimento; questo rendimento è legato a doppio filo con il nostro intelletto.

Author: Ho tolto il termine "verticale" perché può risultare fuorviante.

Per il punto uno, invece, credo che la frase sia più che comprensibile ad una rilettura, si fa infatti riferimento alle considerazioni che ognuno di noi fa, e non alla realtà dei fatti, che viene documentata sotto con uno studio.

> Dal punto di vista della catalogazione l’intelligenza, come capacità di risolvere un problema, viene descritta come Fluida o Cristallizzata (teorizzata dallo psicologo Cattell). La prima, quella fluida, è il pensiero logico, la nostra capacità di risolvere problemi nuovi. Quella cristallizzata, invece, è la nostra capacità di utilizzare le conoscenze pregresse; questa è quella che di solito cade sotto il nome di “seniority”. Nel lavoro dello sviluppatore sono importanti entrambe probabilmente in egual misura.

Reviewer: ```suggestion
Dal punto di vista della catalogazione, l’intelligenza, come capacità di risolvere un problema, viene descritta come Fluida o Cristallizzata (teorizzata dallo psicologo Cattell). La prima, quella fluida, è il pensiero logico, la nostra capacità di risolvere problemi nuovi. Quella cristallizzata, invece, è la nostra capacità di utilizzare le conoscenze pregresse; questa è quella che di solito cade sotto il nome di “seniority”. Nel lavoro dello sviluppatore sono importanti entrambe probabilmente in egual misura.
```

> Dal punto di vista della catalogazione, l’intelligenza, come capacità di risolvere un problema, viene descritta come Fluida o Cristallizzata (teorizzata dallo psicologo Cattell). La prima, quella fluida, è il pensiero logico, la nostra capacità di risolvere problemi nuovi. Quella cristallizzata, invece, è la nostra capacità di utilizzare le conoscenze pregresse; questa è quella che di solito cade sotto il nome di “seniority”. Nel lavoro dello sviluppatore sono importanti entrambe probabilmente in egual misura.

Reviewer 5: ```suggestion
Dal punto di vista della catalogazione, l’intelligenza, come capacità di risolvere un problema, viene descritta come Fluida o Cristallizzata (teorizzata dallo psicologo Cattell). La prima, quella fluida, è il pensiero logico, la nostra capacità di risolvere problemi nuovi. Quella cristallizzata, invece, è la nostra capacità di utilizzare le conoscenze pregresse; questa è quella che di solito cade sotto il nome di “seniority”. Nel lavoro di chi sviluppa sono importanti entrambe probabilmente in egual misura.
```

> Il passare del tempo va intuitivamente beneficio dell'intelligenza cristallizzata perché ci da un bacino più ampio a cui attingere. In passato questa catalogazione veniva legata al Q.I. da cui prende anche il nome: era il rapporto tra età psicologica e anagrafica (da cui quoziente). L’intelligenza fluida, invece, ha una caratteristica inversa, cioè con l’età ha una tendenza a calare, facendoci perdere freschezza. Le nostre performance come velocità, capacità di visualizzazione ed elaborazione sono legate a questi due elementi che sommati formano il nostro intelletto.

Reviewer: typo 

```suggestion
Il passare del tempo va intuitivamente a beneficio dell'intelligenza cristallizzata perché ci da un bacino più ampio da cui attingere. In passato questa catalogazione veniva legata al Q.I. da cui prende anche il nome: rappresentato dal rapporto tra età psicologica e anagrafica (da cui quoziente). L’intelligenza fluida, invece, ha una caratteristica inversa, segue un andamento decrescente con l'avanzare dell'età, facendoci perdere freschezza. Le nostre performance come velocità, capacità di visualizzazione ed elaborazione sono legate a questi due elementi che sommati formano il nostro intelletto.
```

> Il passare del tempo va intuitivamente a beneficio dell'intelligenza cristallizzata perché ci da un bacino più ampio da cui attingere. In passato questa catalogazione veniva legata al Q.I. da cui prende anche il nome: rappresentato dal rapporto tra età psicologica e anagrafica (da cui quoziente). L’intelligenza fluida, invece, ha una caratteristica inversa, segue un andamento decrescente con l'avanzare dell'età, facendoci perdere freschezza. Le nostre performance come velocità, capacità di visualizzazione ed elaborazione sono legate a questi due elementi che sommati formano il nostro intelletto.

Reviewer 5: ```suggestion
Il passare del tempo va intuitivamente a beneficio dell'intelligenza cristallizzata perché ci dà un bacino più ampio da cui attingere. In passato questa catalogazione veniva legata al Q.I. da cui prende anche il nome, rappresentato dal rapporto tra età psicologica e anagrafica (da cui quoziente). L’intelligenza fluida, invece, ha una caratteristica inversa, segue un andamento decrescente con l'avanzare dell'età, facendoci perdere freschezza. Le nostre performance come velocità, capacità di visualizzazione ed elaborazione sono legate a questi due elementi che sommati formano il nostro intelletto.
```

> Da questo punto di vista non stiamo dicendo nulla di nuovo perché è un sentire comune ed è da qui che a volte emerge la volontà di avanzare nella carriera allontanandosi dalla parte operativa che è considerata più avida del primo tipo di intelligenza. Con l’avanzare delle decadi, ad ogni passo, si fa sempre più pressante questa preoccupazione, ma la domanda che ci dobbiamo porre è quanto questa discesa incida sul serio e quanto, invece, possa essere culturale.

Reviewer: ```suggestion
Da questo punto di vista non stiamo dicendo nulla di nuovo perché è un sentire comune ed è da qui che a volte emerge la volontà di avanzare nella carriera allontanandosi dalla parte operativa che è considerata più avida nei confronti del primo tipo di intelligenza. Con l’avanzare delle decadi, ad ogni passo, questa preoccupazione si fa sempre più pressante, ma la domanda che ci dobbiamo porre è quanto questa discesa incida sul serio e quanto, invece, possa avere un impatto culturale.
```

> Da questo punto di vista non stiamo dicendo nulla di nuovo perché è un sentire comune ed è da qui che a volte emerge la volontà di avanzare nella carriera allontanandosi dalla parte operativa che è considerata più avida nei confronti del primo tipo di intelligenza. Con l’avanzare delle decadi, ad ogni passo, questa preoccupazione si fa sempre più pressante, ma la domanda che ci dobbiamo porre è quanto questa discesa incida sul serio e quanto, invece, possa avere un impatto culturale.

Reviewer 2: Non ho capito la scelta delle parole:
- "avida"
- "impatto culturale"

Reviewer 5: Non mi è chiaro l'intento dell'intero paragrafo

Author: Rileggendolo può risultare poco chiaro, me ne rendo conto, l'ho rimosso e a breve committo.

> Gli studi più recenti a riguardo sono stati svolti dall’Università di Monaco, racchiusi in un articolo dal titolo ["Life cycle patterns of cognitive performance over the long run" di Anthony Strittmatter](https://www.pnas.org/doi/10.1073/pnas.2006653117) che ha studiato e cercato di dare una risposta quantitativa e non solo qualitativa alla domanda: _quanto è differente un cervello giovane da uno vecchio e quale è il periodo in cui rende di più?_.

Reviewer 6: Aggiungerei un ndr dove dici l'anno in cui è stata scritta questa cosa. Adesso sono gli ultimi studi fatti ma chissà quando qualcuno leggerà questo capitolo (magari motiva anche qualcuno a cercare qualcosa di più recente se vuole contribuire)

Reviewer 6: Mi riferisco alla frase:

> Gli studi più recenti [...]

Il punto di partenza è stato quello di capire quale sia il metodo migliore per misurare questa performance senza farsi influenzare. Il campione in questo caso è fondamentale: cultura, classe sociale e posizione geografica possono influenzare il risultato ed è semplicemente intuibile; l’allenamento delle proprie capacità e la possibilità di stare vicino a chi detiene conoscenze avanzate danno sempre strumenti migliori per affrontare i problemi. Da questo lato la sfida è molto interessante perché sappiamo bene quanto un bias cognitivo possa influenzare non solo i risultati ma anche la loro interpretazione.

> Per questa analisi si è quindi deciso di usare gli scacchi come metro; le partite, infatti, sono catalogate dal 1890 e la vittoria è legata a processi cognitivi prettamente meritocratici dove tutti quei parametri in cui il dato può essere influenzato hanno meno peso. Il bacino si è presentato piuttosto ampio, sono a disposizione tutte le mosse in un arco di 125 anni, circa 24.000 partite. A questo punto sono stati elaborati collegando età e qualità delle mosse e la velocità con cui venivano eseguite, una ad una; per analizzare la validità delle mosse è stato usato un famoso motore di scacchi open source STOCKFISH.

Reviewer 6: ```suggestion
Per questa analisi si è quindi deciso di usare gli scacchi come metro; le partite, infatti, sono catalogate dal 1890 e la vittoria è legata a processi cognitivi prettamente meritocratici dove tutti quei parametri in cui il dato può essere influenzato hanno meno peso. Il bacino si è presentato piuttosto ampio, sono a disposizione tutte le mosse in un arco di 125 anni, circa 24.000 partite. A questo punto sono stati elaborati collegando età e qualità delle mosse e la velocità con cui venivano eseguite, una ad una; per analizzare la validità delle mosse è stato usato un famoso motore di scacchi open source Stockfish.
```

Il risultato è piuttosto disarmante, è stato notato innanzitutto che il picco di performance è tra i 35 e i 45 anni. Questo va un poco a sfatare il fatto che la giovane età sia legata a performance migliori in assoluto. Da questo punto di vista si nota come l’avanzare dell’età non faccia che migliorare la qualità delle nostre capacità per molti anni; questo chiaramente da un punto di vista prettamente statistico, picchi in tutte le direzioni sono possibili e sono quelli che spesso danno forza ai nostri bias. I dati hanno anche mostrato una cosa altrettanto interessante sugli estremi di questa curva: la quantità di performance guadagnata e poi persa con l’età. Ci troviamo di fronte ad una campana schiacciata, prima sale (dai 20) e poi scende (verso i 55). L’escursione in ambo i lati, ovvero nei 20 e poi nei 55, è solamente intorno al 10%.

> In buona sostanza, quello che si mostra, è che la nostra volontà di fare carriera non deve essere influenzata dalla preoccupazione di non farcela ad una certa età; per lo meno non deve esserlo dal punto di vista delle capacità intellettive. Statisticamente non c’è un vero e proprio declino cognitivo, ma ci si trova più che altro di fronte alla stanchezza o alla propria ambizione di cambiamento. Molti informatici famosi hanno sostenuto questa affermazione: _la carriera non deve essere intesa nella sola direzione manageriale_; se chi si trova a dover prendere questa scelta ha la voglia e la passione, può continuare a fare questo lavoro a vita, non c’è qualcosa che veramente ce lo neghi dal punto di vista fisiologico. Che carriera intraprenderemo quindi deve essere frutto di una passione, di una necessità che non sia legata alla paura di invecchiare; anche questo tema sarà sicuramente rivisto negli anni a venire anche dal punto di vista del mercato del lavoro.

Reviewer: ```suggestion
In buona sostanza, ciò che è evidente è che la nostra volontà di fare carriera non deve essere influenzata dalla preoccupazione di non farcela al sopravanzare di una certa età; per lo meno non deve esserlo dal punto di vista delle capacità intellettive. Statisticamente non c’è un vero e proprio declino cognitivo, ma ci si trova più che altro di fronte alla stanchezza o alla propria ambizione di cambiamento. Molti informatici famosi hanno sostenuto questa affermazione: _la carriera non deve essere intesa nella sola direzione manageriale_; se chi si trova a dover prendere questa scelta ha la voglia e la passione, può continuare a fare questo lavoro a vita, non c’è qualcosa che veramente ce lo neghi dal punto di vista fisiologico. Che carriera intraprenderemo quindi deve essere frutto di un desiderio, di una necessità che non sia legata alla paura di invecchiare; anche questo tema sarà sicuramente rivisto negli anni a venire anche dal punto di vista del mercato del lavoro.
```

> **Se vi piace programmare, avere le mani in pasta ed essere operativi, non fatevi spaventare dell’avanzare dell’età, è più un bias che un ostacolo tangibile.**

Reviewer 2: ```suggestion
**Se vi piace programmare e avere le mani in pasta ed essere operativi, non fatevi spaventare dell’avanzare dell’età: è più un bias che un ostacolo tangibile!**
```
